### PR TITLE
BUGFIX: BB-2305 Remove temporary measure format override CHANGELOG

### PR DESCRIPTION
**Pull request by tomassatinsky**
---

**Proposed release type:** major|minor|patch (see points 6. - 8. of the [semver spec](https://semver.org/#semantic-versioning-specification-semver))

# PR checklist

- [ ] Change was tested using dev-release in Analytical Designer and Dashboards (if applicable).
- [ ] Change is described in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Migration guide (for major update) is written to [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] The proposed release type is appropriate (see the comment in [PR template](../blob/master/.github/pull_request_template.md))

### CODE REVIEW

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#618](https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/618/))
Covered lines: **86.19%**, change: **0%**
Covered branches: **78.86%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/621/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/174/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/621/ : SUCCESS in 59s

**Review by scavnickyj — APPROVED** [review]

**Comment by scavnickyj** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,18 +13,29 @@ The REST API versions in the table are just for your information as the values a
 |\>= 10.0.0|3
 |<= 9.0.1|2
 
-<a name="13.2.0"></a>
-## 2020-08-04 Version [13.1.3](https://github.com/gooddata/gooddata-js/compare/v13.1.1...v13.2.0)
+<a name="13.1.4"></a>
+## 2020-10-16 Version [13.1.3](https://github.com/gooddata/gooddata-js/compare/v13.1.3...v13.1.4)
```
Comment (scavnickyj) [inline CHANGELOG.md:17]: Version [13.1.4]

**Review by scavnickyj — APPROVED** [review]

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#622](https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/622/))
Covered lines: **86.19%**, change: **0%**
Covered branches: **78.86%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/623/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/175/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/623/ : SUCCESS in 1m 00s

**Comment by yenkins** [comment]
## The code coverage is **the same** after your change 🤟
This is a good thing if you are not sure ;)
#### Report (compared to build [#622](https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/622/))
Covered lines: **86.19%**, change: **0%**
Covered branches: **78.86%**, change: **0%**
#### Coverage report
https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/624/artifact/ci/results/coverage/lcov-report/index.html

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-fossa-scanning-zuul-el7/21/ : FAILURE in 1m 53s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/148/ : SUCCESS in 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/624/ : SUCCESS in 1m 08s

**Pull request merged**